README.md

```diff
@@ -131,9 +131,9 @@ https://wiki.openmrs.org/display/docs/OpenMRS+SDK
 
 OpenMRS has a modular architecture that allows developers to extend the OpenMRS core functionality by creating modules that can easily be added or removed to meet the needs of a specific implementation.
 
-Before creating your own module go to the [OpenMRS Module Repository](https://addons.openmrs.org/) and see if there is already a module for your specific use case. If so deploy and try it and if a functionality is missing join the developers of the module to add a feature.
+Before creating your own module go to the [OpenMRS Module Repository](https://addons.openmrs.org/) and see if there is already a module for your specific use case. If so deploy and try it and if functionality is missing join the developers of the module to add a feature.
```

Comment:
thanks @MoulinaPradhan  we can also add a comma after if so around for example (if so,

Reply:
I have committed the changes.